devel/ansible/roles/bodhi/tasks/main.yml

```diff
@@ -109,6 +109,21 @@
       name: jinja2-cli
       executable: pip3
 
+- name: pip install WebOb-GraphQL
```

Comment:
here you can use a loop in ansible instead of repeating the tasks 4 times. See https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#iterating-over-a-simple-list